wgs/wasm/deliverables/wasm-oci-artifact/index.md

```diff
@@ -109,14 +109,15 @@ running the a component if it can not satisfy the imports.
 - Static files: Virtual file system - https://github.com/bytecodealliance/wasi-virt?tab=readme-ov-file#filesystem or specialized component that could be provided to do further customization
 
 ## Implementations
-No current implementations but the following projects have interest:
 
-- containerd/runwasi
-- Warg
-- Spin
+The following projects have implemented this layout (alphabetical order):
+
+- cargo-component
+- containerd
```

Comment:
runwasi has support in its [image building](https://github.com/containerd/runwasi/tree/main/crates/oci-tar-builder#wasm-artifact-usage) but containerd doesn't have support for pulling yet.

Could we split this into two parts `implementations` and `interested` projects? or something to that affect?  We can add https://github.com/bytecodealliance/componentize-dotnet/issues/28 to interested projects

Reply:
I'm fine if the implementations are partial so I think we're ok there. I'll add back in interested projects

Reply:
containerd as a project all up is different from the shim, just want to make sure we are not speaking for core maintainers of containerd.  But as an aside, I guess it does have some basic support since we can pull the image in latest version of containerd:

```
sudo ./bin/ctr i pull ghcr.io/jsturtevant/wasi-artifact:latest
ghcr.io/jsturtevant/wasi artifact:latest        saved
└──manifest (55183e5822e0)                      already exists
   ├──unknown (0db51ed1c948)                    already exists
   └──unknown (52ec249f08df)                    already exists
application/vnd.oci.image.manifest.v1+json sha256:55183e5822e0ebdb5fba6e5a80e432e87aa1f9c23927d4492383be1bad25c562
Pulling from OCI Registry (ghcr.io/jsturtevant/wasi-artifact:latest)    elapsed: 3.7 s     total:  1.9 Mi  (526.1 KiB/s)
```

Reply:
It can also be run via the CRI runtime see https://github.com/containerd/runwasi/pull/661